From rpjday@mindspring.com  Thu Aug  2 18:39:47 2007
From: "Robert P. J. Day" <rpjday@mindspring.com>
Date: Mon, 30 Jul 2007 06:38:31 -0400 (EDT)
Subject: USB: Typo: "USB_SAFE_PADDED" -> "USB_SERIAL_SAFE_PADDED".
To: linux-usb-devel@lists.sourceforge.net
Cc: gregkh@suse.de
Message-ID: <Pine.LNX.4.64.0707300636420.11434@localhost.localdomain>



Fix typo in safe_serial.c to match the actual CONFIG variable.

Signed-off-by: Robert P. J. Day <rpjday@mindspring.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/serial/safe_serial.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/usb/serial/safe_serial.c
+++ b/drivers/usb/serial/safe_serial.c
@@ -74,13 +74,13 @@
 #include <linux/usb/serial.h>
 
 
-#ifndef CONFIG_USB_SAFE_PADDED
-#define CONFIG_USB_SAFE_PADDED 0
+#ifndef CONFIG_USB_SERIAL_SAFE_PADDED
+#define CONFIG_USB_SERIAL_SAFE_PADDED 0
 #endif
 
 static int debug;
 static int safe = 1;
-static int padded = CONFIG_USB_SAFE_PADDED;
+static int padded = CONFIG_USB_SERIAL_SAFE_PADDED;
 
 #define DRIVER_VERSION "v0.0b"
 #define DRIVER_AUTHOR "sl@lineo.com, tbr@lineo.com"
